pps_mw_training/utils/calculate_mean_std.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any
+from pathlib import Path
+import numpy as np  # type: ignore
+import xarray as xr  # type: ignore
+
+
+def get_std_mean(
+    input_files: list[Path], input_params: list[dict[str, Any]]
+) -> list[dict[str, Any]]:
+    """calculate std and mean for the input dataset to normalise"""
+
+    all_dict = {}
+    with xr.open_dataset(input_files[0]) as ds:
+        for key in ds.keys():
+            all_dict[key] = {"n": 0, "x": 0, "x2": 0, "min": 1e10, "max": 0}
+
+    for file in input_files:
+        with xr.open_dataset(file) as ds:
+            ds = ds.sel(
+                {
+                    "npix": ds["nscan"].values[8:-8],
+                    "nscan": ds["npix"].values[8:-8],
+                }
+            )
+            for key in ds.keys():
```

Comment:
is .keys() needed? and would nice to have a better name than key, e.g. parameter